### ISSUE TITLE
browserify-shim as a dependency?

### ISSUE THREAD

**Issue by Stenerson** [davidchin/react-input-range#16]
When installing through npm, importing the module and building I received the error `Error: Cannot find module 'browserify-shim' from...`

I'm not sure why this would be required but perhaps it needs to be moved from devDependencies to  dependencies? I installed `browserify-shim` myself and It started building fine.

Maybe something on my end is messed up but I thought I'd add it here just in case. Thoughts?

**Comment by rohandas** [comment]
I faced the same issue, except even after installing browserify-shim I got the error `Label.js:74 Uncaught TypeError: Cannot read property 'Component' of undefined`, and am not sure how to solve that.

**Comment by Stenerson** [comment]
@rohandas not sure if this is your issue or not but I've seen this before when using `var InputRange = require('react-input-range');` on components that use es6 exports. Perhaps using `import InputRange from 'react-input-range';` will fix this for you?

**Comment by tomnewport** [comment]
Just echoing same problem as @rohandas - I'm using ES6 imports:

```
import InputRange from 'react-input-range';
```

```
    Label.js:77 Uncaught TypeError: Cannot read property 'Component' of undefined
    at Object.<anonymous> (Label.js:77)
    at Object.223 (Label.js:86)
    at s (_prelude.js:1)
    at _prelude.js:1
    at Object.<anonymous> (Slider.js:25)
    at Object.224../Label (Slider.js:252)
    at s (_prelude.js:1)
    at _prelude.js:1
    at Object.<anonymous> (InputRange.js:25)
    at Object.222../Label (InputRange.js:773)
```

**Comment by davidchin** [comment]
Thanks for raising this issue. The problems listed above should be fixed with `^1.0.0`.
